scripts/cmake/version.cmake

```diff
@@ -59,6 +59,24 @@ if(NOT SOF_TAG)
 	set(SOF_TAG 0)
 endif()
 
+execute_process(COMMAND git ls-files src/*
```

Comment:
Can you inline comment what this is doing here and why.

Reply:
added